resources/assets/icons/success.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 11.9 10.3"><path d="M1.397 6.304l1.9 2.6c.2.3.6.5 1 .3.1 0 .1 0 .1-.1h0c.1 0 .1-.1.2-.2l5.9-7.5" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
we already have a `checkmark` and `checkmark-thin` and this is just another variation of it. Check with oleg if it should replace the `checkmark-thin` one as that one is similar in shape